tests/PKSim.Tests/IntegrationTests/DatabaseUpdateSpecs.cs

```diff
@@ -2,224 +2,42 @@
-         halfLifeLiverParameterFor("CYP2C8").DeviationParameter.Value.ShouldBeEqualTo(19 * 60);
-      }
-
-      [Observation]
-      public void standard_deviation_for_CYP2E1_should_match_predefined_value()
-      {
-         halfLifeLiverParameterFor("CYP2E1").DeviationParameter.Value.ShouldBeEqualTo(19 * 60);
-      }
-   }
-
-   public class When_checking_the_changed_in_the_database_for_version_6_1 : concern_for_DatabaseUpdate
-   {
-      [Observation]
-      public void should_update_the_default_value_of_n_cells_per_incubuation_to_1000()
-      {
-         var simulationActiveProcessRepository = IoC.Resolve<ISimulationActiveProcessRepository>();
-         var process = simulationActiveProcessRepository.ProcessFor("HepatocytesRes");
-         process.Parameter(CoreConstantsForSpecs.Parameter.NUMBER_OF_CELLS_PER_INCUBATION).Value.ShouldBeEqualTo(1000);
-      }
-   }
-
-   public class When_checking_the_changes_in_the_database_for_version_6_3 : concern_for_DatabaseUpdate
-   {
-      [Observation]
-      public void should_set_plasma_protein_scale_factor_variable_in_population()
-      {
-         var paramValueRepo = IoC.Resolve<IParameterValueRepository>();
-         var plasmaProteinScaleFactor = paramValueRepo.All().First(p => p.ParameterName.Equals(CoreConstants.Parameter.PLASMA_PROTEIN_SCALE_FACTOR));
-
-         plasmaProteinScaleFactor.CanBeVariedInPopulation.ShouldBeTrue();
-      }
-
-      [Observation]
-      public void should_add_concentration_in_feces_observer_for_all_models()
-      {
-         var observersRepo = IoC.Resolve<IFlatModelObserverRepository>();
-
-         var observerName = CoreConstants.Observer.CONCENTRATION_IN_FECES;
-         observersRepo.Contains(CoreConstants.Model.FourComp, observerName).ShouldBeTrue();
-         observersRepo.Contains(CoreConstants.Model.TwoPores, observerName).ShouldBeTrue();
-      }
-
-      [Observation]
-      public void should_extend_specific_binding_reaction_with_free_concentration()
-      {
-         var processRepo = IoC.Resolve<ISimulationActiveProcessRepository>();
-         var specBinding = processRepo.ProcessFor("SpecificBinding");
-
-         var aliases = from path in specBinding.Formula.ObjectPaths select path.Alias;
-         aliases.ShouldContain("K_water");
-      }
-
-      [Observation]
-      public void should_set_protein_ontogeny_factors_to_variable()
-      {
-         var flatMoleculesRepo = IoC.Resolve<IFlatMoleculeRepository>();
-         var flatMoleculeMapper = IoC.Resolve<IFlatMoleculeToMoleculeBuilderMapper>();
-
-         var protein = flatMoleculeMapper.MapFrom(flatMoleculesRepo.FindBy(QuantityType.Protein), new FormulaCache());
-
-         protein.Parameter(CoreConstants.Parameter.ONTOGENY_FACTOR_GI).CanBeVaried.ShouldBeTrue();
-         protein.Parameter(CoreConstants.Parameter.ONTOGENY_FACTOR).CanBeVaried.ShouldBeTrue();
-      }
-   }
-
-   public class When_checking_the_changes_in_the_database_for_version_6_4 : concern_for_DatabaseUpdate
```

Comment:
There are not removed. I just flipped the order so that the latest tests are first. Otherwise one always have to scroll to see the latest changes. Or do you mean that I removed a class accidentally?